.solhint.json

```diff
@@ -23,6 +23,10 @@
         ],
         "compiler-fixed": true,
         "no-simple-event-func-name": true,
-        "not-rely-on-time": false
+        "not-rely-on-time": false,
+        "no-empty-blocks": "off",
+        "two-lines-top-level-separator": "warn",
+        "separate-by-one-line-in-contract": "warn",
+        "bracket-align": "warn"
     }
-}
+}
```

Comment:
And the war of new line at EOF continues....

Reply:
Another place where I don't like the auto-changes imposed by this